lib/Sema/TypeCheckAttr.cpp

```diff
@@ -6026,6 +6026,16 @@ void AttributeChecker::visitNonisolatedAttr(NonisolatedAttr *attr) {
       }
     }
 
+    // We do not support 'nonisolated' on wrapped properties, because
+    // the backing property is a 'var' and so we cannot mark it as
```

Comment:
This sentence is a bit tautological how about:

```
Using 'nonisolated' with wrapped properties is unsupported, because backing storage
in a stored 'var' that is part of the internal state of the actor which could only be accessed
in actor's isolation context.
```